src/main/java/com/rarchives/ripme/ripper/rippers/NewsfilterRipper.java

```diff
@@ -0,0 +1,92 @@
+package com.rarchives.ripme.ripper.rippers;
+
+
+import com.rarchives.ripme.ripper.AlbumRipper;
+import org.jsoup.Connection;
+import org.jsoup.Jsoup;
+import org.jsoup.nodes.Document;
+import org.jsoup.nodes.Element;
+import org.jsoup.select.Elements;
+
+import java.io.IOException;
+import java.net.MalformedURLException;
+import java.net.URL;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class NewsfilterRipper extends AlbumRipper {
+    private static final String HOST = "newsfilter";
+
+    public NewsfilterRipper(URL url) throws IOException {
+        super(url);
+    }
+
+    @Override
+    public boolean canRip(URL url) {
+        //http://newsfilter.org/gallery/he-doubted-she-would-fuck-on-cam-happy-to-be-proven-wrong-216799
```

Comment:
Could you update the source to provide a SFW link to test with? I'd prefer to keep NSFW links and content out of the code as much as is reasonable.